plugins/taper/run.go

```diff
@@ -211,17 +212,20 @@ func (a *Agent) createTelemetry(item *api.OutputChannelItem) (*models.Telemetry,
 
 	pathAndQuery := common.GetPathWithQuery(request.URL)
 
+	host, _ := common.GetHostAndPortFromURL(request.URL.String(), a.config.TapperNamespace)
+	destinationNamespace := common.GetDestinationNamespaceFromHostOrDefault(host, a.config.TapperNamespace)
+
```

Comment:
`a.config.TapperNamespace` is not the great default namespace to use I think.
We need to use the client namespace so if the host does not hold `.namespace` we can assume that the service is in the same namespace of the **client** and not the tapper.

Reply:
You right